### DISCUSSION TITLE
Best opensource model tested on PandasAI ?

### DISCUSSION THREAD

**Discussion by octadion** [sinaptik-ai/pandas-ai#1035] [Q&A]
Is there anyone here using an open-source model? If so, could you share the name of the model and how accurate it is? Does the model support multiple languages, not just English? thank you in advance.

**Comment by cboettig** [comment]
So far I have had the best experience with llama3.1 (self-hosted via the ollama connection), but have found it is still considerably less effective than, say, `gpt-4o-mini`.  I'd be curious to hear more about others experiences as well.
